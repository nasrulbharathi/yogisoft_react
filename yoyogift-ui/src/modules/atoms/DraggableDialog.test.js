import React from "react";
import { shallow } from "enzyme";
import DraggableDialog from "./DraggableDialog";
test("should render DraggableDialog", () => {
  const wrapper = shallow(<DraggableDialog />);
  expect(wrapper).toMatchSnapshot();
});

describe("Test DraggableDialog function ", () => {
  it("testing handleOpenClose cardvalue", () => {
    const wrapper = shallow(<DraggableDialog />);
    const instance = wrapper.instance();
    instance.handleOpenClose();
    expect(wrapper.state().open).toEqual(true);
  });
  it("testing handleSendAndClose cardvalue", () => {
    let prop = {
      getEmail() {},
    };
    const wrapper = shallow(<DraggableDialog {...prop} />);
    const instance = wrapper.instance();
    instance.handleSendAndClose("handleSendAndClose");
    expect(wrapper.state().open).toEqual(false);
  });
});
