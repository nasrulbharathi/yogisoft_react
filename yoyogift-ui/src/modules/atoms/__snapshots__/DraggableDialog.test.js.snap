// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render DraggableDialog 1`] = `ShallowWrapper {}`;
